nav2_costmap_2d/costmap_plugins.xml

```diff
@@ -1,17 +1,21 @@
 <class_libraries>
-  <library path="liblayers">
+  <library path="layers">
     <class type="nav2_costmap_2d::InflationLayer"  base_class_type="nav2_costmap_2d::Layer">
       <description>Inflates obstacles to speed collision checking and to make robot prefer to stay away from obstacles.</description>
     </class>
+    <!--
```

Comment:
Does leaving these in cause things to break?

Reply:
I'm not sure, I'll make sure it doesn't break and uncomment.